site/styles/_footer.scss

```diff
@@ -21,14 +21,19 @@ footer {
 
 .footer-nav {
   margin-bottom: 30px;
+  vertical-align: middle;
 
   @include media-breakpoint-up(xl) {
-    margin: 0 50px 20px;
+    display: flex;
+    justify-content: space-between;
+    margin: 0 50px;
   }
 
   .nav {
+    display: flex;
     flex-direction: column;
     justify-content: center;
+    justify-content: space-between;
```

Comment:
This doesn't appear to have any effect... probably delete it?

Reply:
fixed